src/components/bulk-selection-snackbar/BulkSelectionSnackbarAction.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { SBulkSelectionSnackbarAction } from "./styles/SBulkSelectionSnackbarAction";
+
+interface IBulkSelectionSnackbarAction {
+  onClick: () => void;
+}
+
+export const BulkSelectionSnackbarAction: React.FC<IBulkSelectionSnackbarAction> = ({ ...props }) => {
```

Comment:
@nicolasshiken Prefer using destructuring for props. Use named imports for React pices.